### DISCUSSION TITLE
how to build libPluginBase.dll on windows

### DISCUSSION THREAD

**Discussion by jcy1001** [alibaba/ilogtail#690] [Help] [answered]
how to build libPluginBase.dll on windows

**Comment by henryzhx8** [comment]
编译插件系统就能得到这个

**Comment by messixukejia** [comment]
看看有没有编译报错

**Comment by jcy1001** [comment]
when build release failed，error like this
when build debug failed


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alibaba/ilogtail"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月3日(星期五) 上午7:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alibaba/ilogtail] how to build libPluginBase.dll on windows (Discussion #690)





 
看看有没有编译报错

**Comment by messixukejia** [comment, marked as answer]
关注下 https://github.com/alibaba/ilogtail/pull/745
